src/components/device/ha-device-trigger-picker.ts

```diff
@@ -0,0 +1,155 @@
+import "@polymer/paper-icon-button/paper-icon-button";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-item/paper-icon-item";
+import "@polymer/paper-item/paper-item-body";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu-light";
+import "@polymer/paper-listbox/paper-listbox";
+import memoizeOne from "memoize-one";
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  css,
+  CSSResult,
+  property,
+} from "lit-element";
+import { HomeAssistant } from "../../types";
+import { fireEvent } from "../../common/dom/fire_event";
+import { compare } from "../../common/string/compare";
+import { LocalizeFunc } from "../../common/translations/localize";
+import computeStateName from "../../common/entity/compute_state_name";
+
+export interface DeviceTrigger {
+  platform: string;
+  domain: string;
+  device_id: string;
+  entity_id?: string;
+  type: string;
+}
+
+const fetchDeviceTriggers = (hass, deviceId) =>
```

Comment:
Move this to `data/device_automation.ts`, the interface too.

Reply:
Fixed.